src/WebUI/Components/Common.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+import { BaseComponent } from "@uifabric/utilities/lib";
+import { IStatusProps, Status, StatusSize } from "azure-devops-ui/Status";
+import { IVssComponentProperties } from "../Types";
+
+export interface IPodStatusProps extends IVssComponentProperties {
+    statusProps?: IStatusProps;
+    statusSize: StatusSize;
+    statusDescription?: string;
+    customDescription?: React.ReactNode;
+}
+
+export class PodStatusComponent extends BaseComponent<IPodStatusProps, {}> {

```

Comment:
i would recommend this to be under components/common/somecomponent.tsx, that way it is easy to know, otherwise we may bloat this file with other components also

Reply:
Moved to a file of its own: PodStatusComponent.tsx